Frontend/frontend/src/Util/APIRequest.js

```diff
@@ -0,0 +1,27 @@
+import User from "./User";
+
+const apiBaseUrl = process.env.REACT_APP_BACKEND_API
+
+export default class APIRequest {
+    static async login(username, password) {
```

Comment:
Again, could be good to have a docstring or something similar to explain what the class does since we are expected to have comments in our code.